graphql/resolvers.go

```diff
@@ -1789,6 +1789,42 @@ func (r *queryResolver) TaskTests(ctx context.Context, taskID string, execution
 	}, nil
 }
 
+func (r *queryResolver) TaskTestSample(ctx context.Context, tasks []string, testFilters []*TestFilter) ([]*TaskTestResultSample, error) {
+	if len(tasks) == 0 {
+		return nil, nil
+	}
+	t, err := task.Find(task.ByIds(tasks))
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("finding task %s: %s", tasks[0], err))
```

Comment:
nit: printing only tasks[0] may be misleading for errors originating from the variant history page where we are checking multiple tasks.

Reply:
```suggestion
		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error finding task %s: %s", tasks[0], err))
```